src/ui/style/global.module.css

```diff
@@ -83,6 +83,7 @@ body.pageLayout {
   --body-width: 100%;
   height: 100vh;
   width: 100%;
+  border: none;
```

Comment:
Not sure, but looks like it is not related to global shortcuts

Reply:
right, it's a also a bug I found due to this refactoring: https://github.com/zeriontech/zerion-wallet-extension/pull/523